File: MasterAndroid/Pages/01.ViewBinding/app/src/main/java/com/viewbinding/MainActivity.java
package com.viewbinding;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.viewbinding.databinding.ActivityMainBinding;
public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
    @Override
    public void onStart() {
        super.onStart();
        binding.textViewActivity.setText(getString(R.string.hello_from_vb_activity));
    }
}

File: MasterAndroid/Books/AndroidProgramming/01.ExploringLayouts/app/src/main/java/com/magicbytes/exploringlayouts/MainActivity.kt
package com.magicbytes.exploringlayouts

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_menu)
    }

    fun loadConstraintLayout(v: View) {
        setContentView(R.layout.activity_main)
    }

    fun loadTableLayout(v: View) {
        setContentView(R.layout.my_table_layout)
    }

    fun loadMenuLayout(v: View) {
        setContentView(R.layout.main_menu)
    }
}